LiveSplit.AutoSplitters.xml

```diff
@@ -4734,7 +4734,7 @@
             <URL>https://raw.githubusercontent.com/LukeSaward1/AutoSplitters/main/Ursa/Ursa_Autosplitter.asl</URL>
         </URLs>
         <Type>Script</Type>
-        <Description>Game Time and Auto Start/Split/Reset are available. (By LukeSaward)</Description>
+        <Description>Game Time and Auto Start/Split/Reset are available. (By LukeSaward & Ero)</Description>
```

Comment:
The ampersand is the problem, you need to write it as &amp; as it's treated specially in XML.